locale/ca/docs/index.md

```diff
@@ -25,20 +25,7 @@ També descriu els mòduls inclosos que proporciona Node.js, mes no documenta el
 
 ### Buscant la referència de l'API per a una versió anterior?
 
-* [Node.js 15.x](https://nodejs.org/docs/latest-v15.x/api/)
-* [Node.js 13.x](https://nodejs.org/docs/latest-v13.x/api/)
-* [Node.js 12.x](https://nodejs.org/docs/latest-v12.x/api/)
-* [Node.js 11.x](https://nodejs.org/docs/latest-v11.x/api/)
-* [Node.js 10.x](https://nodejs.org/docs/latest-v10.x/api/)
-* [Node.js 9.x](https://nodejs.org/docs/latest-v9.x/api/)
-* [Node.js 8.x](https://nodejs.org/docs/latest-v8.x/api/)
-* [Node.js 7.x](https://nodejs.org/docs/latest-v7.x/api/)
-* [Node.js 6.x](https://nodejs.org/docs/latest-v6.x/api/)
-* [Node.js 5.x](https://nodejs.org/docs/latest-v5.x/api/)
-* [Node.js 4.x](https://nodejs.org/docs/latest-v4.x/api/)
-* [Node.js 0.12.x](https://nodejs.org/docs/latest-v0.12.x/api/)
-* [Node.js 0.10.x](https://nodejs.org/docs/latest-v0.10.x/api/)
-* [Totes les versions](https://nodejs.org/docs/)
+{#var nodeVersionLinks}* [Totes les versions](https://nodejs.org/docs/)
```

Comment:
If you rename the template to `.hbs`, I think you can use `{{> html-head }}`. and move this one translatable string of the last bullet into the regular `site.json`

Reply:
@nschonni: I've removed this md file as a data (just like versions.json) into "source" folder, and refer it into md to be rendered with.

Reply:
Sorry, I've been trying to figure it out locally too. Not sure if it's a limitation of the Metalsmith, or if I'm also not getting the syntax right

Reply:
@nschonni: Nothing, I thought you'd something I didn't know about XD, so I ask you again and again...

Reply:
Maybe https://github.com/fidian/metalsmith-handlebars-contents would work